test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
```

Comment:
We don't use test folder, so untrack them
1. Open `.gitignore` and add `test` line
2. Run `git rm --cached -r test`

Reply:
done